awx/main/access.py

```diff
@@ -933,8 +934,12 @@ class CredentialAccess(BaseAccess):
      - I'm a superuser.
      - It's a user credential and it's my credential.
      - It's a user credential and I'm an admin of an organization where that
-       user is a member of admin of the organization.
+       user is a member.
+     - It's a user credential and I'm a credential_admin of an organization
+       where that user is a member.
      - It's a team credential and I'm an admin of the team's organization.
+     - It's a team credential and I'm a credential admin of the team's
+       organization.
```

Comment:
Both of these claims sound untrue to me. Filtering the credential list is done strictly by implicit membership to the credential's read_role. A "private" user credential does not have an organization IIRC, which means that members of the organization credential admin role wouldn't be able to see them.